src/release.js

```diff
@@ -592,16 +598,42 @@ module.exports = function taoExtensionReleaseFactory(params = {}) {
 
             // Get last released version:
             await gitClient.pull(releaseBranch);
-            const { version: lastVersion } = await this.getMetadata();
-            data.lastVersion = lastVersion;
-            data.lastTag = `v${lastVersion}`;
-
-            // Get version to release:
             await gitClient.pull(baseBranch);
-            const manifest = await this.getMetadata();
-            data.version = manifest.version;
-            data.tag = `v${manifest.version}`;
-            data.releasingBranch = `${branchPrefix}-${manifest.version}`;
+
+            const lastTag = await gitClient.getLastTag();
+
+            let {
+                recommendation: {
+                    hasNonConventionalCommits,
+                },
+                lastVersion,
+                version,
+            } = await conventionalCommits.getNextVersion(lastTag);
+
+            if (releaseVersion && !semverGt(releaseVersion, lastVersion)) {
+                log.exit(`Provided version less than latest version ${lastVersion}.`);
+            }
+
+            if (!releaseVersion && hasNonConventionalCommits) {
+                const { pull } = await inquirer.prompt({
+                    type: 'confirm',
+                    name: 'pull',
+                    message: 'There are some non conventional commits?. Are you sure you want to continue?',
+                });
```

Comment:
It would be great to bypass such confirmation message, the final goal is automation

Reply:
Should it be just a warn message instead? Be aware of that without this prompt user will not be able to abort release in case if there is no conventional commits

Reply:
If we want a fully automated version without confirmations, couldn't we activate that mode via a CLI parameter?

Reply:
as i understand there will be another task to have possibility to make fully automated releases. And it should cover not only prompts, it will need to handle some cases which currently done manually